### ISSUE TITLE
TypeError: token.type.endsWith is not a function when having multiline v-show statement with `!` on the start of one of the lines.

### ISSUE THREAD

**Issue by ndyakov** [mysticatea/vue-eslint-parser#25]
Although I submitted this to `eslint-plugin-vue` as the plugin throws the error I think this is related to the parser.
```
├─┬ eslint-plugin-vue@4.2.2
│ └── vue-eslint-parser@2.0.2 

```
https://github.com/vuejs/eslint-plugin-vue/issues/396

**Comment by mysticatea** [comment]
Thank you for this report.

However, this is not related to this parser. It's a but in `babel-eslint`.
